test/c/test_vf2_layout.c

```diff
@@ -69,9 +69,9 @@ static int test_vf2_layout_line(void) {
     QkVF2LayoutConfiguration *layout_config = qk_vf2_layout_configuration_new();
     qk_vf2_layout_configuration_set_call_limit(layout_config, 10000, 10000);
     QkVF2LayoutResult *layout_result =
-        qk_transpiler_pass_standalone_vf2_layout(qc, target, layout_config, false);
+        qk_transpiler_pass_standalone_vf2_layout_average(qc, target, layout_config, false);
```

Comment:
For consistency with the newly-added tests, would it make sense to rename this and the other existing test as `test_vf2_average_<...>`?

Reply:
Sure, done in 80ce20b15db3cb4106ae7e3985e56ec47c1168d6.